README.md

```diff
@@ -1,5 +1,5 @@
 # Porfolio-mobile-version-skeleton
-Personal Porfolio from
+Personal Porfolio from Microverse
 In this repository, with guidance from Microverse, we will create a complete porfolio from sratchg (Mobile and Desktop Version)
 
```

Comment:
- [ ] As this is a pair programming project where both of you contribute to this project, It is important to give credit to your partner as an author in the **authors section** so please add your partner in the README file 😃 👍